search/search-client-connect/src/main/java/io/camunda/search/connect/configuration/Database.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.search.connect.configuration;
+
+public final class Database {
+  public static final String ElasticSearchString = "elasticsearch";
+  public static final String RdbmsString = "rdbms";
+  public static final String OpenSearchString = "opensearch";
+
+  public enum Type {
```

Comment:
:wrench: Why not just have a single top level type `DatabaseType`? Also would be great if I can get the enum value from a string, as this will be a common case.